### PR TITLE
[Glimmer2] Fix some tests related input

### PR DESCRIPTION
**Pull request by chadhietala**

### CODE REVIEW

File: packages/ember-views/lib/mixins/text_support.js [hunk 1]
```diff
@@ -196,9 +197,12 @@ export default Mixin.create(TargetActionSupport, {
   },
 
   _elementValueDidChange() {
-    // Using readDOMAttr will ensure that HTMLBars knows the last
-    // value.
-    set(this, 'value', this.readDOMAttr('value'));
+    if (isEnabled('ember-glimmer')) {
```
Comment (krisselden) [inline packages/ember-views/lib/mixins/text_support.js]: curious, why did you need to guard this? what case does this break in htmlbars?

File: packages/ember-testing/lib/helpers/fill_in.js [hunk 2]
```diff
@@ -11,10 +10,10 @@ export default function fillIn(app, selector, contextOrText, text) {
   $el = app.testHelpers.findWithAssert(selector, context);
   el = $el[0];
   focus(el);
-  run(() => {
```
Comment (krisselden) [inline packages/ember-testing/lib/helpers/fill_in.js]: we should cleanup any other test helpers that are nesting run loops in the wait one.

Comment (chadhietala) [inline packages/ember-testing/lib/helpers/fill_in.js]: I'll follow up with that.

Comment (chadhietala) [inline packages/ember-testing/lib/helpers/fill_in.js]: See #13836
